static/app/views/starfish/components/starfishProjectSelector.tsx

```diff
@@ -34,6 +49,7 @@ export function StarfishProjectSelector() {
 
   return (
     <CompactSelect
+      disabled={!projectsLoaded}
```

Comment:
```suggestion
```

Small nit, but looks like this will never be disabled so this prop isn't necessary. `if(!projectsLoaded)` is true, it will just return the other compact select.

Reply:
True, I had this added earlier when I was trying a different approach and forgot to remove it 👍